src/shogun/kernel/MaternKernel.h

```diff
@@ -0,0 +1,78 @@
+/*
+ * This software is distributed under BSD 3-clause license (see LICENSE file).
+ *
+ * Authors: Gil Hoben
+ */
+
+#include <shogun/kernel/DistanceKernel.h>
+#include <shogun/kernel/Kernel.h>
+
+
+namespace shogun {
+
+
+	/**
+	 * @brief the class Matérn kernel 
+	 *
+	 * It is defined as
+	 * \f[
+	 * k(x, y) = \frac{2 ^ {\nu - 1}}{\Gamma(\nu)}\Bigg(\sqrt{2\nu}\frac{D(x, y)}{\rho}\Bigg)^\nu + N_{\nu}\Bigg(\sqrt{2\nu}\frac{D(x, y)}{\rho}\Bigg)
+	 * \f]
+	 * Where:
+	 * 		\f$N_{\nu}\f$ is the Bessel function of the second kind with order \f$\nu\f$,
+	 *		\f$\rho\f$ is the kernel width, and
+	 *		\f$\Gamma\f$ is the gamma function
+	 **/
+	class MaternKernel: public DistanceKernel 
```

Comment:
@karlnapf do you know from what class I should inherit from?

Reply:
translation invariant kernel?

Reply:
yea, thought so, will change it